website/docs/components/segmented-group/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,15 @@
+## Conformance rating
+
+<Doc::Badge @type="warning">Conditionally conformant</Doc::Badge>
+
+You must make sure each `TextInput` or `Select` segment has an accessible name, either by associating them with a `<label>` element, or by setting an `aria-label` on each input control.
+
+## Applicable WCAG Success Criteria
+
+This section is for reference only. This component intends to conform to the following WCAG Success Criteria:
+
+<Doc::WcagList @criteriaList={{array "1.3.1" "1.3.2" "1.3.4" "1.3.5" "1.4.1" "1.4.3" "1.4.4" "1.4.10" "1.4.11" "1.4.12" "2.4.6" "2.4.7" "3.2.1" "3.2.2" "3.2.4" "3.3.1" "3.3.2" "4.1.1" "4.1.2" }} />
+
+---
+
+<Doc::A11ySupport />
```

Comment:
@MelSumner I added the criteria I found for the input/select+button, but do let me know if you think it needs adjustments. Likewise for the conformance rating.

Reply:
That seems like a reasonable approach to me. 👍